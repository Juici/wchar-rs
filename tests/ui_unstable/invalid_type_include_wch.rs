use wchar::include_wch;

const INVALID_TYPE: &[f32] = include_wch!(f32, "../data/basic.txt");

fn main() {}
