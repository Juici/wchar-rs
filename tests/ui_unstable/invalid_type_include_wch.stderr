error: unexpected wchar type `f32`, expected one of: u16, u32, i16, i32
 --> $DIR/invalid_type_include_wch.rs:3:43
  |
3 | const INVALID_TYPE: &[f32] = include_wch!(f32, "../data/basic.txt");
  |                                           ^^^
